dot/network/host_test.go

```diff
@@ -330,12 +330,11 @@ func TestStreamCloseMetadataCleanup(t *testing.T) {
 	require.NoError(t, err)
 
 	const (
-		roles           byte   = 4
 		bestBlockNumber uint32 = 77
 	)
 
 	testHandshake := &BlockAnnounceHandshake{
-		Roles:           roles,
+		Roles:           common.Validator,
```

Comment:
nit it would be nice to have a `roles` package somewhere (maybe in the common directory?), so you could call it using `roles.Validator` instead

Reply:
I don't think it is a good idea to have a package just for one struct, which is barely 10 lines of code. I could however rename this to `common.ValidatorRole` to improve readability.

Reply:
done

Reply:
this is more of a discussion than something to do here -

I agree I don't like 10 lines packages; but `common` / `utils` / `shared` packages are bad package names and they usually bundle a mix bag of things. Not something we may want to do anytime soon, but having meaningful package names would be nice.